bin/main

```diff
@@ -1,111 +1,130 @@
 #!/usr/bin/env ruby
-def disply_board(array)
-  position = 0
-  while position < array.length
-    if (position % 3).zero?
-      puts ''
-      print "---+---+---+\n"
+require_relative '../lib/logic'
+class Game
+  attr_reader :board, :move, :tic_tac_toe
+  attr_accessor :player1, :player2, :players, :loc, :locations, :usedd
```

Comment:
- [ ] The student uses attr_readers/attr_writers. Only uses attr_accessors if they really need it and must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Modify this to attr_reader and attr_writers unless you are sure that you need accessors please.